snuba/subscriptions/data.py

```diff
@@ -79,6 +79,7 @@ class SubscriptionData:
     time_window_sec: int
     entity_subscription: EntitySubscription
     query: str
+    organization: Optional[int] = None
```

Comment:
```suggestion
    org_id: Optional[int]
```
- org_id is the actual name of the column, let's align to this
- remove the default value, i think it should be explicitly passed
- i would probably probably put this directly below project_id

Reply:
Sure, it's also important to note that `organization` is the default key in the data payload. For processes (e.g. tests) that both encode and decode its own payload, we would need to support parsing for both `organization` and `org_id` then.

Reply:
Yeah I think this is important to note. There is no change to how the data is being stored (or there shouldn't be). Before this change, the org ID was stored in the `organization` key in the dictionary.

I would advise against changing anything in how the data is actually stored. That's a risk we don't need to take.